workers/nintendo/Dockerfile

```diff
@@ -81,7 +106,8 @@ RUN apt-get update && \
 
 ENV DEVKITPRO=/opt/devkitpro
 ENV DEVKITPPC=$DEVKITPRO/devkitPPC
-ENV PATH=$PATH:$DEVKITPPC/bin:$DEVKITPPC/powerpc-eabi/bin
+ENV DEVKITARM=$DEVKITPRO/devkitARM
+ENV PATH=$PATH:$DEVKITPPC/bin:$DEVKITPPC/powerpc-eabi/bin:$DEVKITARM/bin:$DEVKITARM/arm-none-eabi/bin
```

Comment:
This won’t work right if the build code is well-behaved, since the DS compiler will pick up package configurations from devkitPPC first. It’s probably best to just split the DS and Wii into two workers if we’re going to keep supporting DS.